exporters/ostream/src/metric_exporter.cc

```diff
@@ -79,13 +79,34 @@ sdk::common::ExportResult OStreamMetricExporter::Export(
 
   for (auto &record : data.scope_metric_data_)
   {
-    printInstrumentationInfoMetricData(record);
+    printInstrumentationInfoMetricData(record, data);
   }
   return sdk::common::ExportResult::kSuccess;
 }
 
+void OStreamMetricExporter::printAttributes(
+    const std::unordered_map<std::string, sdk::common::OwnedAttributeValue> &map,
+    const std::string prefix)
+{
+  for (const auto &kv : map)
```

Comment:
@bsarden - Looks like this part of the code is causing problem. As it's unordered_map, the order of printing of resource elements is implementation specific ( so different in Windows and Linux). Can we ensure to print them in the given order - maybe move elements to std::map and then print? And also ensure that the same order is used for comparison in unit-test.

Reply:
@bsarden  - Just wondering if you have a plan to fix CI as mentioned above? Else, I can do it on top of your repo if you are fine :)

Reply:
Thanks for fixing it. CI looks good now :)

Reply:
Thanks for the ping and for reproducing on windows!

Reply:
Thanks for the ping and reproducing the error on a windows box!